Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Footer.css';
+
+
+const Footer = () => {
+    return (<div className='footer-container'>
+    	       <i className="fa fa-comment-o fa-lg" ariaHidden="true"></i>
+    	       <i className="fa fa-refresh fa-lg" ariaHidden="true"></i> 
+    	       <i className="fa fa-heart-o fa-lg" ariaHidden="true"></i>
+    	       <i className="fa fa-envelope-o fa-lg" ariaHidden="true"></i>           
```

Comment:
Consider a different naming pattern for your classes. This can get confusing very quickly in a large app.